aad/spring-security/reactive/webflux/oauth2/spring-cloud-gateway/gateway/src/main/java/com/azure/spring/sample/activedirectory/reactive/webflux/oauth2/gateway/gateway/configuration/SecurityConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package com.azure.spring.sample.activedirectory.reactive.webflux.oauth2.gateway.gateway.configuration;
+
+import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity;
+
+@EnableWebFluxSecurity
+public class SecurityConfiguration {
```

Comment:
what's the difference with ApplicationConfiguration, can we put the @EnableWebFluxSecurity in the same class?